src/frontend/apps/standalone_site/src/features/Playlist/components/PlaylistPage.tsx

```diff
@@ -15,6 +17,28 @@ import { PlaylistOrderType, usePlaylists } from '../api/usePlaylists';
 
 import { CreatePlaylistForm } from './CreatePlaylistForm';
 
+function isTouchDevice() {
+  return 'ontouchstart' in window || navigator.maxTouchPoints > 0;
+}
+
+const ContainerBox = styled(Box)`
+  ${isTouchDevice()
+    ? ``
+    : `
+  button {
+    opacity: 0;
+    transition: .4s;
+  }
+
+  :hover {
+    button {
+      opacity: 1;
+      transition: .4s;
+    }
+  }
+  `}
+`;
```

Comment:
I feel like this kind of conditional construction may cause problems when switching to cunningham…
WDYT ?

Reply:
Why would it ? 
It will just style a div (or the cunningham equivalent) rather than a Box imo.

Reply:
My point is it could be hard to spot (and delete) when we will replace the buttons.
But maybe I'm wrong…